cindy/index.js

```diff
@@ -0,0 +1,43 @@
+//These exercises are aimed at making you familiar with logical operators. Print all your checks to the console. Every capitalized word stands for the logical operator that you must use in each task. Make sure that you are using the correct logical operator.
+
+//Declare two variables: "dogIsBetter" with the value of true, and "catIsBetter" with the value of false.
+let dogIsBetter = true;
+let catIsBetter = false;
+
+//Check the result of:
+
+//a) dogIsBetter AND catIsBetter.
+console.log(dogIsBetter && catIsBetter);
+
+//b) dogIsBetter OR catIsBetter.
+console.log(dogIsBetter || catIsBetter);
+
+//c) NOT (dogIsBetter AND catIsBetter)
+console.log(!(dogIsBetter && catIsBetter));
+
+//Declare three more variables "atoms", "sandGrains", "starsInSky". Give these variables number values.
+let atoms = 10;
+let sandGrains = 20;
+let starsInSky = 5;
+
+//a) atoms is greater than starsInSky AND atoms is greater than sandGrains.
+console.log("a " + (atoms > starsInSky > sandGrains));
```

Comment:
3a- The answer is incorrect! Please try my solution instead!
console.log( (atoms > starsInSky) && (atoms > sandGrains) );

Reply:
done it